### ISSUE TITLE
Tap Notes

### ISSUE THREAD

**Issue by pul53gh** [redrockruby/ch-flat-mod#3]
This may or not be intentional but the theme seems to be using the unused tap note, not an edited theme-matching note.

![image](https://user-images.githubusercontent.com/75389344/119377705-6a9bbf00-bc8b-11eb-9c19-c040e49c7e71.png)

**Comment by solamint** [comment]
I wasn't intending to do the alternate tap notes, but I can add it in.

**Comment by pul53gh** [comment]
> I wasn't intending to do the alternate tap notes, but I can add it in.

Yeah I was wondering why it was the alt-tap and not an edited texture.
